target/linux/ath79/image/generic.mk

```diff
@@ -607,6 +607,15 @@ define Device/engenius_ecb1750
 endef
 TARGET_DEVICES += engenius_ecb1750
 
+define Device/engenius_ens202ext
+  ATH_SOC := ar9341
```

Comment:
This typop causes
```
mips-openwrt-linux-musl-cpp: error: ../dts/_engenius_ens202ext.dts: No such file or directory
mips-openwrt-linux-musl-cpp: warning: '-x assembler-with-cpp' after last input file has no effect
mips-openwrt-linux-musl-cpp: fatal error: no input files
compilation terminated.
```

This should be: `SOC := ar9341`, without `ATH_` prefix!

Reply:
indeed.

Reply:
Ah yes, another artifact from tag v19.07.2, thanks.

I'm going to build and test from master once we decide on the switch situation, and also looking into LOADER_FLASH_OFFS